db/migrate/20171026145935_group_assignment_uniqueness_group_offer_combined_with_volunteer.rb

```diff
@@ -0,0 +1,7 @@
+class GroupAssignmentUniquenessGroupOfferCombinedWithVolunteer < ActiveRecord::Migration[5.1]
+  def change
+    add_column :group_assignments, :active, :boolean, default: true
```

Comment:
for what is this needed in the group_assignments? isn't it already handled in the group offer itself?

Reply:
That is the recommended way, to do it as well in the db.

Reply:
What is meant by active?

Reply:
This is working with paranoia gem, same as with the Users table. This way only not softdeleted records are checked for uniqueness. deleted records are not throwing an error if they'd duplicate with a not soft deleted one

Reply:
the field name is given by paranoiagem